src/main/resources/db/migration/migrate_data_from_director_v1.sql

```diff
@@ -0,0 +1,155 @@
+--SET @@sql_mode = CONCAT(@@sql_mode, ',', 'ONLY_FULL_GROUP_BY');
+--SET @@sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
+
+-- move this to old director code
+create index if not exists ecu_serial_idx on director.ecus (`ecu_serial`);
+create index if not exists ecu_update_assignments_target_v1_idx on director.ecu_update_assignments (`filepath`(500), `length`(254), checksum);
+
+-- repo_namespaces
+INSERT director_v2.repo_namespaces
+SELECT * FROM director.repo_names v1
+ON DUPLICATE KEY UPDATE repo_id = v1.repo_id, created_at = v1.created_at, updated_at = v1.updated_at;
+
+-- select count(*) from repo_namespaces ;
+
+DROP TEMPORARY TABLE IF EXISTS ecu_targets_v1;
+create temporary table ecu_targets_v1 AS
+select namespace, max(id) id, filepath filename, length, max(checksum) checksum, hash sha256, uri, max(created_at) created_at, max(updated_at) updated_at from (
+select namespace, uuid() id, filepath, length, checksum, json_unquote(json_extract(checksum, '$.hash')) hash, NULL uri, created_at, updated_at
+from director.current_images c
+UNION
+select namespace, uuid(), target filepath, target_size length, JSON_OBJECT("method", "sha256", "hash", target_hash), target_hash, target_uri uri, created_at, updated_at
```

Comment:
Is it `target filepath` or `target filename`? I might be missing something, but the definition for `ecu_targets` only has `filename` and I don't see more references to this `filepath`.

Reply:
in director v2 is filename, filepath is just an alias here because that is what old director calls it. so we give everything the same name so we can SELECT on in in line 18.